Makefile

```diff
@@ -0,0 +1,24 @@
+
+################
+# General Jobs #
+################
+
+default: vue
+
+vue: prep
+	NPM_CMD=dev docker-compose up; docker-compose down
```

Comment:
We should hold off on adding docker-compose down. Right now anybody using this will likely need to add some test db data to develop against so we should let them clean up their containers explicitly when required.

Reply:
@stephenhillier Db persists in ./.tmp.  I'll add a db clean to the Makefile in the AM.

Reply:
@stephenhillier +fixtures?  What else?